packages/icon/package.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os, subprocess
+from spack import *
+
+
+class Icon(AutotoolsPackage):
+    """The ICON modelling framework is a joint project between the
+    German Weather Service and the
+    Max Planck Institute for Meteorology for
+    developing a unified next-generation global numerical weather prediction and
+    climate modelling system. The ICON model has been introduced into DWD's
+    operational forecast system in January 2015."""
+
+    homepage = 'https://software.ecmwf.int/wiki/display/GRIB/Home'
```

Comment:
There is not really an official homepage anymore. Most things are handled done via https://gitlab.dkrz.de/icon/icon.

Reply:
Fixed